plugins/eth_swap_plugin/eth_swap_plugin.cpp

```diff
@@ -381,7 +390,7 @@ namespace eosio {
     }
 
     void eth_swap_plugin::plugin_startup() {
-        ilog("Ethereum swap plugin started. Version 2.0");
+        ilog("Ethereum swap plugin started. Version 2.0.1");
```

Comment:
either pull version from the build or remove it completely.

Reply:
Done.